io_functions/io_func.c

```diff
@@ -0,0 +1,79 @@
+#include <stdio.h>
+
+//#define IN 1 // inside word
```

Comment:
Please remove commented code if it's not needed anymore or add more context if it has to be kept.